src/coreclr/vm/field.h

```diff
@@ -133,7 +133,6 @@ class FieldDesc
 
     enum {
         enum_packedMbLayout_MbMask        = 0x01FFFF,
```

Comment:
Can this and `RequiresFullSlotNumber` be deleted as well?

Reply:
I left them both since they represent a bit more complex logic that we might want to reuse. However, they both can be removed as they serve no purpose at present.

Reply:
Removed